src/App.css

```diff
@@ -0,0 +1,17 @@
+  /*main {
+    margin-top: 50px;
+  }*/
+  form {
+    margin-bottom: 10px;
+  }
+  form-inline {
+    width: 100%;
+  }
+  ul {
+    margin-bottom: 0px;
+  }
```

Comment:
extra lines in the end of file, remove them.
You can use some code editor like vscode for formatting and other things. + it has a lot of packages for different languages support, etc.